From 6e4c682972061bb05e7f49078f01d7b856fe298d Mon Sep 17 00:00:00 2001
From: dana <dana@dana.is>
Date: Fri, 13 Mar 2020 09:46:14 -0500
Subject: [PATCH] unposted: _git: Fix copy/paste error in earlier commit that
 broke `git stash drop`

This was broken by workers/44923 (8aa84bd5fc44)

Originally: https://github.com/zsh-users/zsh/commit/754658aff38e1bdf487c58bec6174cbecd019d11

Signed-off-by: Michael Hoang <enzime@users.noreply.github.com>
---
 Completion/Unix/Command/_git | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Completion/Unix/Command/_git b/Completion/Unix/Command/_git
index 97ab26512..675460dad 100644
--- a/Completion/Unix/Command/_git
+++ b/Completion/Unix/Command/_git
@@ -1809,7 +1809,7 @@ _git-stash () {
           _nothing
           ;;
         (drop)
-          _arguments -S $endopt \_git-notes
+          _arguments -S $endopt \
             '(-q --quiet)'{-q,--quiet}'[suppress all output]' \
             ':: :__git_stashes' && ret=0
           ;;
-- 
2.33.0

